msteams-platform/bots/how-to/conversations/Workflow bot in Teams.md

```diff
@@ -0,0 +1,460 @@
+---
+title: Workflow bot in Teams
+description: Learn how to send a response to card action in Teams workflow bot, add more card actions and customize action responses.
+ms.topic: conceptual
+ms.author: surbhigupta
+ms.localizationpriority: high
+---
+
+# Workflow bot in Teams
+
+A workflow bot allows users to interact with an Adaptive Card, enabled by the Adaptive Card action handler feature. You can create a workflow bot in many scenarios for your end users like incident management, ticketing, approval workflow, and project management cards. The other simplified use cases are creating and assigning a work item with workflow bot and syncing the content to Azure DevOps/Jira system. A workflow bot can be installed into a team, group chat, or as personal app, depending on different scopes. The default command logic returns an Adaptive Card. You can customize this logic with your business requirement. For the customization, you need to call your existing APIs.
+
+**Advantages of creating a workflow bot**:
+
+1. Automates business processes and repetitive workflows without leaving the context of conversations.
+1. Supports users with sequential workflow through various cards progressively, without sending additional cards.
+1. Provides user-specific views, and is always up-to-date.
+1. Simplifies programming model with TeamsFx SDK.
+
+   > [!NOTE]
+   > You can choose the installation target when adding the app. For more install options, see [distribute your Teams app](../../../concepts/deploy-and-publish/apps-publish-overview.md).
+
+Users can create a workflow bot to respond to Adaptive Card that is triggered by end users. Adaptive Card action handler provides one or more buttons in the card to ask for user's input by calling APIs. The APIs send another Adaptive Card in conversation to respond to the respective card action.
```

Comment:
Can we tweak this a little bit? e.g. 
=> Adaptive Card action handler powered by TeamsFx SDK can handle the adaptive card universal action (`Action.Execute`) that triggered by end users, and thenl send another Adaptive Card in conversation to respond to the respective card action.

Reply:
Please see the following rephrasing of the suggestion you've give:
Adaptive Card action handler powered by TeamsFx SDK can execute the Adaptive Card universal action (`Action.Execute`) that is triggered by end users. In response to this card action in the conversation another Adaptive Card is sent.

Reply:
I've changed as above. Kindly confirm.